### ISSUE TITLE
showing games in find search

### ISSUE THREAD

**Issue by cnsaavedra** [cnsaavedra/old-mmo-friends#3]
after "resolving" the Cors issue, we are presented with a new issue that results with "undefined".
Postman manages to get the result we wanted though.

**Comment by cnsaavedra** [comment]
Okay, so I figured out the problem. In a way, I did not have to deal with any of the cors stuff because I believe that was not the main source of the problem.

So, instead of accessing my database data through "GET" and getting the data by doing this:
```javascript
this.data = (Service.function(bodyValue)).data 
```
I did "POST" to get the data, and accessed the data by simply doing this
```javascript
const response = Service.function({
                    id: bodyValue
                })
this.data = response.data
```
This accesses the data without having to get "secured" information from the database, but by accessing the data from the database by getting Observer object info from the database.

The Observer object looks as ![follows](https://i.gyazo.com/2c866acd9ce688a01a75adf580599b51.png) , which treats the user data as an object instead of pure data.

Compared to a data ![object](https://i.gyazo.com/52b179bd6c22a7f03f6886d92599415f.png), where each data {...} has user information.

I am not sure if I am using the correct words, but these are to the extent of my current understanding.

**Comment by cnsaavedra** [comment]
Okay, so I figured out the problem. In a way, I did not have to deal with any of the cors stuff because I believe that was not the main source of the problem.

So, instead of accessing my database data through "GET" and getting the data by doing this:
```javascript
this.data = (Service.function(bodyValue)).data 
```
I did "POST" to get the data, and accessed the data by simply doing this
```javascript
const response = Service.function({
                    id: bodyValue
                })
this.data = response.data
```
This accesses the data without having to get "secured" information from the database, but by accessing the data from the database by getting Observer object info from the database.

The Observer object looks as ![follows](https://i.gyazo.com/2c866acd9ce688a01a75adf580599b51.png) , which treats the user data as an object instead of pure data.

Compared to a data ![object](https://i.gyazo.com/52b179bd6c22a7f03f6886d92599415f.png), where each data {...} has user information.

I am not sure if I am using the correct words, but these are to the extent of my current understanding.
